### PR TITLE
OSDOCS-8382: Migrate "Getting Started with ROSA: Autoscaling" from rosaworkshop.io into product documentation

### PR DESCRIPTION
**Pull request by jneczypor**
[OSDOCS-8382](https://issues.redhat.com//browse/OSDOCS-8382): Migrate "Getting Started with ROSA: Autoscaling" from rosaworkshop.io into product documentation

Version(s):
4.14+

Issue:
https://issues.redhat.com/browse/OSDOCS-8382

Link to docs preview:
https://69847--ocpdocs-pr.netlify.app/openshift-rosa/latest/cloud_experts_tutorials/cloud-experts-getting-started/cloud-experts-getting-started-autoscaling

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by jneczypor** [comment]
Hi @yuwang-RH! May I please get a QE review of this PR? Thanks!

File: cloud_experts_tutorials/cloud-experts-getting-started/cloud-experts-getting-started-autoscaling.adoc [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+:context: cloud-experts-getting-started-autoscaling
+
+toc::[]
+
+//rosaworkshop.io content metadata
+//Brought into ROSA product docs 2024-01-04
+
+The xref:../../rosa_cluster_admin/rosa_nodes/rosa-nodes-about-autoscaling-nodes.adoc#rosa-nodes-about-autoscaling-nodes[cluster autoscaler] adds or removes worker nodes from a cluster based on pod resources.
+
+The cluster autoscaler increases the size of the cluster when:
```
Comment (EricPonvelle) [inline cloud_experts_tutorials/cloud-experts-getting-started/cloud-experts-getting-started-autoscaling.adoc:14]: These bulleted lists do not render correctly:
image

I believe the fix is adding a new line between the line ending with a colon and the first bulleted item. So, line 20 and 21 become 20, 21, and 22:

The cluster autoscaler decreases the size of the cluster when:

* Some nodes are consistently not needed for a significant period. For example, when a node has low resource use and all of its important pods can fit on other nodes.

File: cloud_experts_tutorials/cloud-experts-getting-started/cloud-experts-getting-started-autoscaling.adoc [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+The cluster autoscaler does not increase the cluster resources beyond the limits that you specify. 
+
+The cluster autoscaler decreases the size of the cluster when:
+* Some nodes are consistently not needed for a significant period. For example, when a node has low resource use and all of its important pods can fit on other nodes.
+
+== Enabling autoscaling for an existing machine pool using the CLI
+
+[NOTE]
+====
+Cluster autoscaling can be enabled at cluster creation and when creating a new machine pool by using `--enable-autoscaling`.
```
Comment (EricPonvelle) [inline cloud_experts_tutorials/cloud-experts-getting-started/cloud-experts-getting-started-autoscaling.adoc:27]: s/by using `--enable-autoscaling`./by using the `--enable-autoscaling` option.

It renders weird on a full screen:
image

File: cloud_experts_tutorials/cloud-experts-getting-started/cloud-experts-getting-started-autoscaling.adoc [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+====
+
+. Autoscaling is set based on machine pool availability. To find out which machine pools are available for autoscaling, run the following command:
++
+[source,terminal]
+----
+rosa list machinepools -c <cluster-name>
```
Comment (EricPonvelle) [inline cloud_experts_tutorials/cloud-experts-getting-started/cloud-experts-getting-started-autoscaling.adoc:34]: Add a `$` to the start of this line

File: cloud_experts_tutorials/cloud-experts-getting-started/cloud-experts-getting-started-autoscaling.adoc [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+Cluster autoscaling can be enabled at cluster creation by checking the *Enable autoscaling* checkbox when creating machine pools.
+====
+
+. Go to the *Machine pools* tab and click the three dots in the right..
+. Click *Scale*, then *Enable autoscaling*.
+. Run the following command to confirm that autoscaling was added:
++
+[source,terminal]
+----
+rosa list machinepools -c <cluster-name>
```
Comment (EricPonvelle) [inline cloud_experts_tutorials/cloud-experts-getting-started/cloud-experts-getting-started-autoscaling.adoc:74]: Prepend with `$`

File: cloud_experts_tutorials/cloud-experts-getting-started/cloud-experts-getting-started-autoscaling.adoc [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+[source,terminal]
+----
+rosa edit machinepool -c <cluster-name> --enable-autoscaling <machinepool-name> --min-replicas=<num> --max-replicas=<num>
+----
++
+.Example input
++
+[source,terminal]
+----
+rosa edit machinepool -c my-rosa-cluster --enable-autoscaling Default --min-replicas=2 --max-replicas=4
```
Comment (EricPonvelle) [inline cloud_experts_tutorials/cloud-experts-getting-started/cloud-experts-getting-started-autoscaling.adoc:56]: Prepend with `$`

File: cloud_experts_tutorials/cloud-experts-getting-started/cloud-experts-getting-started-autoscaling.adoc [hunk 6]
```diff
@@ -0,0 +1,83 @@
...
+----
+ID         AUTOSCALING  REPLICAS  INSTANCE TYPE  LABELS     TAINTS    AVAILABILITY ZONES
+Default    No           2         m5.xlarge                           us-east-1a
+----
+
+. Run the following command to add autoscaling to an available machine pool:
++
+[source,terminal]
+----
+rosa edit machinepool -c <cluster-name> --enable-autoscaling <machinepool-name> --min-replicas=<num> --max-replicas=<num>
```
Comment (EricPonvelle) [inline cloud_experts_tutorials/cloud-experts-getting-started/cloud-experts-getting-started-autoscaling.adoc:49]: Prepend with `$`

**Review by EricPonvelle — COMMENTED** [review]
Small things, but it looks good otherwise.

**Pull request merged**